landing-pages/site/content/en/community/_index.html

```diff
@@ -16,10 +16,10 @@
             </p>
             <ul class="ticks-blue mx-auto">
                 <li>
-                    Users list: <a class="list-link" href="mailto:users-subscribe@airflow.apache.org">users-subscribe@airflow.apache.org</a> <small>and <a class="list-link" href="mailto:users-unsubscribe@airflow.apache.org">users-unsubscribe@airflow.apache.org</a> to unsubscribe</small>
+                    Users list; for discussion, news, and information such as release details: <a class="list-link" href="mailto:users-subscribe@airflow.apache.org">users-subscribe@airflow.apache.org</a> <small>and <a class="list-link" href="mailto:users-unsubscribe@airflow.apache.org">users-unsubscribe@airflow.apache.org</a> to unsubscribe</small>
```

Comment:
```suggestion
                    Users list - for discussion, news, and information such as release details: <a class="list-link" href="mailto:users-subscribe@airflow.apache.org">users-subscribe@airflow.apache.org</a> <small>and <a class="list-link" href="mailto:users-unsubscribe@airflow.apache.org">users-unsubscribe@airflow.apache.org</a> to unsubscribe</small>
```
Seems to be more natural, but I'm not a native speaker 😄

Reply:
Yep